databricks_cli/sdk/service.py

```diff
@@ -938,3 +938,13 @@ def reset(self, pipeline_id=None, headers=None):
         _data = {}
 
         return self.client.perform_query('POST', '/pipelines/{pipeline_id}/reset'.format(pipeline_id=pipeline_id), data=_data, headers=headers)
+
+    def stop(self, pipeline_id=None, headers=None):
+        _data = {}
+
+        return self.client.perform_query('POST', '/pipelines/{pipeline_id}/stop'.format(pipeline_id=pipeline_id), data=_data, headers=headers)
+
+    def run(self, pipeline_id=None, headers=None):
```

Comment:
Are these generated from the protos or did you do it manually?

Reply:
Oh, makes sense why the code looks so weird. Didn't realize they were generated. Do you happen to know the command to do that?

Reply:
`bb //api:doc_and_sdk`. In theory you just copy the service.py file that gets generated, but they're out of sync, so just grab the pipelines bits that get generated. I think it's probably going to be exactly what you have.

Reply:
Nice, will confirm that it's the same